modules/qt/hm.nix

```
@@ -2,11 +2,14 @@
   pkgs,
   config,
   lib,
+  osConfig ? null,
   ...
 }:
 {
   options.stylix.targets.qt = {
-    enable = config.lib.stylix.mkEnableTarget "QT" pkgs.stdenv.hostPlatform.isLinux;
+    # only autoenables on NixOS systems. see https://github.com/danth/stylix/issues/933
+    enable = config.lib.stylix.mkEnableTarget "QT" (osConfig != null);
```

Why are we dropping the `pkgs.stdenv.hostPlatform.isLinux` guard:

```suggestion
    enable = config.lib.stylix.mkEnableTarget "QT" (
      pkgs.stdenv.hostPlatform.isLinux && osConfig != null
    );
```